include/avif/avif.h

```diff
@@ -428,30 +428,39 @@ typedef uint32_t avifTransformFlags;
 
 typedef struct avifPixelAspectRatioBox
 {
-    // 'pasp' from ISO/IEC 14496-12:2015 12.1.4.3
+    // 'pasp' from ISO/IEC 14496-12:2022 12.1.4.3
 
     // define the relative width and height of a pixel
     uint32_t hSpacing;
     uint32_t vSpacing;
 } avifPixelAspectRatioBox;
 
+// NOTE: The members of the avifCleanApertureBox struct are declared as uint32_t to match the
+// unsigned int(32) type used in ISO/IEC 14496-12:2022 faithfully. However, ISO/IEC 14496-12:2022
+// 12.1.4.1 clearly interprets these values as signed int(32) and talks about them being strictly
+// positive, positive, or negative. Cast these struct members to int32_t before use. See also the
+// clean aperture extension in the QuickTime File Format:
+// https://developer.apple.com/documentation/quicktime-file-format/clean_aperture
```

Comment:
Another thing we can say in this comment is to encourage people to use the `avifCropRect` struct instead and use the two conversion functions to convert between `avifCropRect` and `avifCleanApertureBox`. Do you think that's a good idea?

Reply:
Yes.